### ISSUE TITLE
Java + Lombok + Kotlin - find usages did not work

### ISSUE THREAD

**Issue by alexey-krylov** [mplushnikov/lombok-intellij-plugin#591]
### Short description
Unable to find fields usages of Lombok'ed Java classes in Kotlin files.

### Expected behavior
Alt+F7 on field should find each usages on this field in Kotlin (also with Java).

### Version information

 * **IDEA Version:** 
IntelliJ IDEA 2018.3.4 (Ultimate Edition)
Build #IU-183.5429.30, built on January 29, 2019
JRE: 1.8.0_152-release-1343-b26 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
 * **JDK Version:** 1.8.0_201
 * **OS Type & Version:** Windows 10, 64 bit
 * **Lombok Plugin Version:** v0.23-2018.3
 * **Lombok Dependency Version:** 1.18.4

### Steps to reproduce

 1. Create project with modules in Java and Kotlin
 2. In Java module create class Foo with lombok @Data annotation
 3. In Kotlin module create some service that uses Foo class attributes from Java module
4. Go to Foo.java and try to find usages for attributes that is used in Kotlin module - nothing will be found

### Sample project
-

### Additional information
-

### Stacktrace
-

**Comment by alexey-krylov** [comment]
As i workaround now i'm using 
@FieldDefaults(level = AccessLevel.PUBLIC)
on all fields, in this case all usages is seen by IDEA

**Comment by alexey-krylov** [comment]
Also i've created issue here:
https://youtrack.jetbrains.com/issue/IDEA-207470

**Comment by xinatcg** [comment]
same issue

**Comment by xinatcg** [comment]
it seem like kotlin use property and lombok generate getter, they are confused in IDEA. I can only add a getter to the field

**Comment by BrokerXu** [comment]
Dear Big man, is the problem solved? Ask for help me!

**Comment by FichteFoll** [comment]
No, it's still an issue. The workaround with setting all fields to public does work, but it also means you have public-accessible fields, obviously.
